website/js/app/controllers/content-product.ng

```diff
@@ -115,6 +115,26 @@ Write-GcsObject -Bucket $bucket -File "test-file.png" -ObjectName "test/test.png
 
 # Download a GCS object to disk.
 Read-GcsObject $bucket "object-name" -OutFile "output-file.png"</pre>
+    <h2>Cloud Storage PowerShell Provider</h2>
+    <p>
+      Cloud Tools for PowerShell includes a PowerShell provider for Google Cloud Storage.
+      This provider allows you to use commands like cd, dir, copy and del to navigate
```

Comment:
Let's used a fixed-width font for terminal commands. (That is surround them with "\<code\>" tags.)

like \<code\>cd\</code\>, \<code\>dir\</code\>, ...